openmeter/streaming/connector.go

```diff
@@ -43,6 +43,8 @@ type Connector interface {
 	ListEventsV2(ctx context.Context, params ListEventsV2Params) ([]RawEvent, error)
 	// ListSubjects lists the subjects that have events in the database
 	ListSubjects(ctx context.Context, params ListSubjectsParams) ([]string, error)
+	// ListGroupByValues lists the group by values that have events in the database
+	ListGroupByValues(ctx context.Context, params ListGroupByValuesParams) ([]string, error)
```

Comment:
could/should this replace `ListSubjects`? Looks like it does the same. Or maybe keep on interface just make it call into `ListGroupByValues`.

Reply:
For ListSubjects meter is not required. We can consolidate it later.